### ISSUE TITLE
Webdriver system property not working on windows 7

### ISSUE THREAD

**Issue by DanielaValero** [galenframework/galen#271]
Hello,

I am configuring galen, I am getting this error message for chrome and IE:
```
java.lang.IllegalStateException: The path to the driver executable must be set by the webdriver.ie.driver system property; for more information, see http://code.google.com/p/selenium/wiki/InternetExplorerDriver. The latest version can be downloaded from http://selenium-release.storage.googleapis.com/index.html
```

This is the test:

```
@@ parameterized
    | browser  |
    | phantomjs|
    | chrome   |
    | ie       |
@@ parameterized
    | deviceName | tags      | size     |
    | Mobile     | mobile    | 320x600  |
    | Tablet     | tablet    | 640x480  |
    | Desktop    | desktop   | 1024x800 |

Page-layout on ${deviceName} device in ${browser} browser
    selenium ${browser} ${baseURL}/page.html ${size}
        check  test/specs/page.spec --include "${tags},all"
```

I am not sure what is the reason of this, do you have an idea?

**Comment by hypery2k** [comment]
see [the docs](http://galenframework.com/docs/reference-working-in-command-line/#Providingsystemproperties) that you have to download the [chromedriver](https://code.google.com/p/selenium/wiki/ChromeDriver), e.g.
```
galen test mytest01.test -Dwebdriver.chrome.driver=/opt/chromedriver/chromedriver)
```

**Comment by DanielaValero** [comment]
Hi,

Yes, that was what I missed, the installation of the drivers.

I followed these steps:

1. Download the drivers, [here the IE driver](http://selenium-release.storage.googleapis.com/index.html) and [here the chrome driver](https://sites.google.com/a/chromium.org/chromedriver/downloads)

2. In Windows, place them under: D:\seleniumDrivers, or a similar place.

3. Add the folder in which you've added the drivers to your PATH environment variable.

Then, the usage of the system properties command is not required, because Selenium will just find them.

In my case, my computer became super slow (I have at the moment set only one test), and I decided to execute the visual test with the default browser of galen, firefox.
